client/my-sites/email/email-providers-comparison/stacked/provider-cards/professional-email-card.tsx

```diff
@@ -127,9 +127,9 @@ const ProfessionalEmailCard = ( props: EmailProvidersStackedCardProps ): ReactEl
 	professionalEmail.formFields = (
 		<NewMailBoxList
 			areButtonsBusy={ addingToCart }
-			hiddenFieldNames={ hiddenFieldNames }
 			initialFieldValues={ { [ FIELD_ALTERNATIVE_EMAIL ]: userEmail } }
 			isInitialMailboxPurchase
+			hiddenFieldNames={ hiddenFieldNames }
```

Comment:
I think this change breaks the order of those props 🙂.

Reply:
@AllTerrainDeveloper Thoughts on this?